src/Microsoft.Docs.Template/Views/Template/BuildItemListPartial.cshtml

```diff
@@ -91,15 +91,19 @@ else if (Model.Style == "cards")
             {
                 className = "panelContent " + Model.ClassName;
             }
+            else
+            {
+                className = Model.ClassName;
+            }
             if (Model.Columns is JValue value)
```

Comment:
@live1206 @928PJY I can see another difference between v2 and v3 here: if `Model.Columns` is undefined
- v2 will take `4` by default and append `cols cols4` to className
- v3 wil not append `cols cols` string to `className`

If we want to make the behavior same with v2, maybe we can merge the if conditions:
`if (Model.Columns is JValue value && int.TryParse(value.ToString(), out var i))`

Reply:
looks good to me

Reply:
@live1206 Updated. Could you please help review again?

Reply:
The code looks good to me, deployed to PPE already, could you help verify the diff is gone? Thanks